### ISSUE TITLE
Probelm z endponintem /sale/offer-events

### ISSUE THREAD

**Issue by patrykpolanski-euvic** [allegro/allegro-api#9151]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**

Od wczoraj od godziny około 14:50 mam problem z endpointem
api.allegro.pl/sale/offer-events
Endpoint zwraca mi losowo status 503 lub 504

przykładowy url
api.allegro.pl/sale/offer-events?limit=100&from=MTcxNzU5MDA0MDg0NDg3Mg
api.allegro.pl/sale/offer-events?limit=100&from=MTcxNzY1Njg4OTU4NzY0Nw

odpowiedz z API
503: {"errors":[{"code":"SERVICE_UNAVAILABLE","message":"Service unavailable.","details":null,"path":null,"userMessage":"Server is unable to respond to your query. Try again in a few minutes."}]}

504: {"errors":[{"code":"ServiceTimeoutException","message":"An error occurred","details":null,"path":null,"userMessage":"Query response time exceeded. Try again."}]}

W czym jest problem?
Czy coś jest nie tak z moimi requestami?
Konta allegro:
Ulv-hedin
Ualazar
ksiegralnia_pl

Proszę o pomoc.
Pozdrawiam Pandres

**Comment by Lukasz-Zurek** [comment]
@patrykpolanski-euvic Potwierdzam występujące problemy. Odpowiedni zespół pracuje nad ich możliwie szybkim wyeliminowaniem. Przepraszamy za te utrudnienia.

**Comment by patrykpolanski-euvic** [comment]
Czy coś już wiadomo kiedy problem zostanie wyeliminowany?
Moja aplikacja coraz bardziej się rozsynchrnizuje co powoduje spore problemy w logistyce.

**Comment by patrykpolanski-euvic** [comment]
Zwrotka : 404
https://api.allegro.pl/sale/product-offers/15837280939
{"errors":[{"code":"OfferNotFoundException","message":"Not Found","details":null,"path":"","userMessage":"Offer 15837280939 does not exist.","metadata":{}}]}

**Comment by Lukasz-Zurek** [comment]
@patrykpolanski-euvic Problem z eventami nie powinien się już powtarzać.

Natomiast przesłany przez Ciebie błąd 404 - to znany u nas problem i w ciągu kolejnych dni powinna wyjść poprawka. 

Przepraszamy za te utrudnienia.

**Comment by patrykpolanski-euvic** [comment]
Czy coś się ruszyło z tematem zwracania 404 w endpoincie `sale/product-offers/{offerId}`?

**Comment by PrzemyslawLukanowski** [comment]
@patrykpolanski-euvic Tak, wprowadziliśmy poprawkę, która powinna eliminować ten problem. Jeśli jednak nadal występuje, prześlij proszę przykładowe trace-id.
